src/test/java/HttpServerTest.java

```diff
@@ -0,0 +1,104 @@
+import org.apache.http.HttpResponse;
+import org.apache.http.client.fluent.Request;
+import org.junit.*;
+import org.mockito.Mockito;
+import org.rapidoid.net.Server;
+import ru.kontur.airlock.EventSender;
+import ru.kontur.airlock.HttpServer;
+import ru.kontur.airlock.ValidatorFactory;
+import ru.kontur.airlock.dto.AirlockMessage;
+import ru.kontur.airlock.dto.EventGroup;
+import org.apache.http.HttpStatus;
+import java.util.HashMap;
+import java.util.Properties;
+
+import static org.mockito.Mockito.*;
```

Comment:
Static import with wildcard must be avoided too. Also, one more thing about it from the [java guides](https://docs.oracle.com/javase/1.5.0/docs/guide/language/static-import.html):
> So when should you use static import? Very sparingly! Only use it when you'd otherwise be tempted to declare local copies of constants, or to abuse inheritance (the Constant Interface Antipattern). In other words, use it when you require frequent access to static members from one or two classes. If you overuse the static import feature, it can make your program unreadable and unmaintainable, polluting its namespace with all the static members you import. Readers of your code (including you, a few months after you wrote it) will not know which class a static member comes from. Importing all of the static members from a class can be particularly harmful to readability; if you need only one or two members, import them individually. Used appropriately, static import can make your program more readable, by removing the boilerplate of repetition of class names.

Reply:
fixed